systemSmartContractsConfig.toml

```diff
@@ -20,25 +15,26 @@
 [ESDTSystemSCConfig]
     BaseIssuingCost = "5000000000000000000" #5 eGLD
     OwnerAddress = "erd1ze6pgefwc7mdx0rhje3h50kkgt26ut3gx8ytezndkz8fjcf96gsqsd7ugw"
-    EnabledEpoch = 272
 
 [GovernanceSystemSCConfig]
-    ProposalCost = "5000000000000000000" #5 eGLD
-    NumNodes     = 500
-    MinQuorum    = 400
-    MinPassThreshold = 300
-    MinVetoThreshold = 50
-    EnabledEpoch = 1000000
+    FirstWhitelistedAddress = "erd1agy0x6qrfmhgvyjrurwxrhf6a2xgcxmgmx5l4dw2w6xpa223zr5s04hkw4"
```

Comment:
we coud have used the multisig address for this: erd1qqqqqqqqqqqqqpgq0gq09608js3gje72v0yzc56k6glc852gl94sgtahk2

Reply:
right :facepalm:

Reply:
done